### PR TITLE
LibWeb+WebContent+Ladybird: Add ability to paste text from clipboard

### PR DESCRIPTION
**Pull request by kalenikaliaksandr**
Text can be pasted by pressing Ctrl/Cmd+V or by using button in the context menu. For now only the Qt client is supported.

### CODE REVIEW

File: Userland/Libraries/LibWeb/Page/EventHandler.cpp [hunk 1]
```diff
@@ -868,6 +868,22 @@ bool EventHandler::handle_keyup(KeyCode key, u32 modifiers, u32 code_point)
...
+    auto active_document = m_browsing_context->active_document();
+    if (!active_document)
+        return;
+    if (!active_document->is_fully_active())
+        return;
+
+    if (auto cursor_position = m_browsing_context->cursor_position()) {
+        active_document->update_layout();
+        m_edit_event_handler->handle_insert(*cursor_position, text);
+        auto length = Utf8View(text).length();
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/Page/EventHandler.cpp:882]: You can use `text.code_points().length()` here

**Review by trflynn89 — COMMENTED** [review]
Cool! 
This is fine since it matches existing behavior. FYI I'm planning on making Page's `EventHandler` handle ctrl/cmd shortcuts in a future patch, so the select/select_all/paste IPCs will go away (well, paste will still have an IPC, but in the other direction).

File: Ladybird/Qt/BrowserWindow.cpp [hunk 2]
```diff
@@ -116,6 +116,12 @@ BrowserWindow::BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::Cook
     edit_menu->addAction(m_select_all_action);
     QObject::connect(m_select_all_action, &QAction::triggered, this, &BrowserWindow::select_all);
 
+    m_paste_action = new QAction("&Paste", this);
```
Comment (trflynn89) [inline Ladybird/Qt/BrowserWindow.cpp:119]: Oh sorry, didn't notice before - I think it makes more sense for "Paste" to go right below "Copy". I don't think I've ever seen a UI that is ordered Copy > Select All > Paste

Comment (kalenikaliaksandr) [reply Ladybird/Qt/BrowserWindow.cpp:113]: fixed!

**Comment by kalenikaliaksandr** [comment]
Red CI is not related:
```
==================================================
Pass: 946, Fail: 1, Skipped: 6, Timeout: 0
==================================================
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/window-scrollTo.html
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/Worker/Worker-echo.html
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/Crypto/SubtleCrypto-generateKey.html
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/Crypto/SubtleCrypto-exportKey.html
Fail: /home/vsts/work/1/s/Tests/LibWeb/Text/input/WebAnimations/animation-properties/replaceState.html
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/WebAnimations/animation-properties/startTime.html
Skipped: /home/vsts/work/1/s/Tests/LibWeb/Text/input/WebAnimations/animation-properties/playbackRate.html
```

**Pull request merged**